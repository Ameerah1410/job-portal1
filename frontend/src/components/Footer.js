import { Box } from "@mui/material";
import React from "react";
import { useTheme } from "@mui/material/styles";

// Footer component: Renders a simple footer with a copyright notice
const Footer = () => {
  // Accessing theme information
  const { palette } = useTheme();

  return (
    <>
      {/* Footer box with specified styling */}
      <Box
        sx={{
          height: "70px",
          bgcolor: palette.secondary.midNightBlue,
          display: "flex",
          justifyContent: "center",
          alignItems: "center",
        }}
      >
        {/* Text component for the copyright notice */}
        <Box component="span" sx={{ color: palette.primary.main }}>
          <p>Â© JOB PORTAL 2023</p>
        </Box>
      </Box>
    </>
  );
};

export default Footer;
