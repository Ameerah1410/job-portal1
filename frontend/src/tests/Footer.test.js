import React from "react";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import Footer from "../components/Footer";

test("renders Footer correctly", () => {
  const { asFragment } = render(<Footer />);
  expect(asFragment()).toMatchSnapshot();
});
