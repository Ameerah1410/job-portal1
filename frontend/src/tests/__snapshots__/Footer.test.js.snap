// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Footer correctly 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-1o9gbcf"
  >
    <span
      class="MuiBox-root css-1bm9945"
    >
      <p>
        Â© JOB PORTAL 2023
      </p>
    </span>
  </div>
</DocumentFragment>
`;
